assets/js/components/ExecutionResults/checksUtils.test.js

```diff
@@ -179,5 +180,29 @@ describe('checksUtils', () => {
 
       expect(health).toBe('warning');
     });
+
+    it('should return true or false if a check is premium or not', () => {
+      const checkIds = [
+        faker.datatype.uuid(),
+        faker.datatype.uuid(),
+        faker.datatype.uuid(),
+        faker.datatype.uuid(),
+      ];
+      const expectedPremiumValues = [
```

Comment:
`it.each` would be more appropriate in my opinion here, instead of having an internal loop

Reply:
Sure, next time i use it.each :D